.github/workflows/flowzone.yml

```diff
@@ -0,0 +1,27 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches: [main, master]
+  # allow external contributions to use secrets within trusted code
+  pull_request_target:
+    types: [opened, synchronize, closed]
+    branches: [main, master]
+
+jobs:
+  flowzone:
```

Comment:
I'm guessing this needs some changes to continue testing on mac/windows, cc @klutchell ?

Reply:
@Page- Flowzone still doesn't support npm tests on macos or windows, it hasn't really been a priority. It needs to be done via custom actions like we did here:

https://github.com/balena-io-modules/winusb-driver-generator/blob/master/.github/workflows/flowzone.yml
https://github.com/balena-io-modules/winusb-driver-generator/blob/master/.github/actions/test/action.yml

I would like to add multiple OS support to npm checks in Flowzone, but it complicates our testing matrix quite a bit and I don't want to slow down the pre-checks. We can open a research/project if it's worth looking into further.